src/plugins/graphql/transforms.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable max-len */
```

Comment:
Is this file used anywhere?

I didn't see it used in this PR ?

Reply:
It's used by `src/plugins/graphql/signature.js` (so basically not any of our files, but the other file coming from `apollo-graphql`).

Reply:
Oh wow it seems a commit is missing, let me update the PR.